scripts/toolboxSrc/pgEnsureExtensions.ts

```diff
@@ -0,0 +1,9 @@
+import {sql} from 'kysely'
+import getKysely from '../../packages/server/postgres/getKysely'
+
+export default async () => {
+  console.log('🔩 Postgres Extension Checks Started')
+  const pg = getKysely()
+  await sql`CREATE EXTENSION IF NOT EXISTS "vector";`.execute(pg)
```

Comment:
is this where we should use the env var?
i suppose we could also just do a try/catch & swallow the error if we want to avoid using an env var? the env var might be useful at runtime though when deciding if we should use embeddings or not. your call!

Reply:
Wait. Lolololo did I seriously not use the ENV var? Ha! Good catch. I will fix